### PR TITLE
Added redirects for mstg

### PR DESCRIPTION
**Pull request by commjoen**

### CODE REVIEW

File: redirects/mstg.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 title: MSTG Redirect
 permalink: /mstg
+redirect_from:
+  - /mstg/
+  - /mstg
```
Comment (ThunderSon) [inline redirects/mstg.md:7]: Since this is the permalink, no need for this one :)

Comment (commjoen) [reply redirects/mstg.md]: fixed, thsanks!

File: redirects/mstg.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 title: MSTG Redirect
 permalink: /mstg
+redirect_from:
+  - /mstg/
+  - /masvs
+  - /masvs/
```
Comment (kingthorin) [inline redirects/mstg.md:8]: Are these both really one project?

Comment (commjoen) [reply redirects/mstg.md:8]: yes :)

**Pull request merged**